.github/workflows/release-cli.yml

```diff
@@ -6,13 +6,17 @@ on:
       - 'v**'
 
 env:
-  VERSION: ${{ github.ref_name }}
```

Comment:
Niektore casti potrebuju verziu bez `v` na zaciatku, takze som to nahradil jobom `version`.